src/main/java/marquez/db/Columns.java

```diff
@@ -35,9 +35,6 @@ private Columns() {}
   // Namespace ownership column names
 
   public static final String NAMESPACE_UUID = "namespace_guid";
-  public static final String STARTED_AT = "started_at";
-  public static final String ENDED_AT = "ended_at";
-  public static final String OWNER_UUID = "owner_uuid";
```

Comment:
this doesn't seem related :(

Reply:
try to keep relevant things in the same PR, otherwise it will seem like `owner_uuid` was somehow part of cleaning up `startedAt` and `endedAt`.

Reply:
Thanks, fixed!